dashboard_ui/config/dashboards/Temperature Monitoring/celsius.json

```diff
@@ -513,8 +514,8 @@
   },
   "timepicker": {},
   "timezone": "",
-  "title": "Temperature_starter_solution",
-  "uid": "70aiGq34k",
-  "version": 6,
+  "title": "Celsius",
+  "uid": "2iNwMfWNk",
```

Comment:
Keep same uid and title as before - otherwise if someone updates they will get three dashboards

Reply:
version should be incremented

Reply:
I will reset the UID and increment the version, but I think giving the dashboard a meaningful title makes sense.